spec/controllers/ncr/work_orders_controller_spec.rb

```diff
@@ -115,6 +115,19 @@
       end
     end
 
+    it "allows the project title to be edited" do
```

Comment:
this is a key test for this PR. Before the changes in this PR, this did not pass (we were using `name` param instead)